portfolio/src/main/webapp/script.js

```diff
@@ -25,9 +25,26 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
-//add Hello Nzinga! to the page
-async function getHelloNzingaUsingAsyncAwait() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('HelloNzinga-container').innerText = quote;
-}
\ No newline at end of file
+
+function getHelloNzingaUsingAsyncAwait() {
+    fetch("data") // sends a request to /my-data-url
+    .then(response =>{
+        console.log(response);
+        return response.json();
+    }) // parses the response as JSON
+    .then((myObject) => { // now we can reference the fields in myObject!
+    var quote= "";
+    
+    myObject.forEach(function(element){
+        console.log(element);
+        var para = document.createElement("p");
+        var node = document.createTextNode(element.toString());
+        para.appendChild(node);
+
+        var element = document.getElementById('HelloNzinga-container');
+        element.appendChild(para);
+    })
+
+});
```

Comment:
[Style](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces. 

Try using the [Prettier](https://docs.google.com/document/d/1yeDuDfEoVh2RB2wC-KrhkXNlekc15RxkPy8p-G9U4tA/edit#heading=h.el4nfk7uwuhr) tool to fix these formatting issues.